packages/website/ts/components/portal/menu.tsx

```diff
@@ -38,49 +49,66 @@ const menuItemEntries: MenuItemEntry[] = [
     },
 ];
 
-const DEFAULT_LABEL_COLOR = colors.darkerGrey;
-const DEFAULT_ICON_COLOR = colors.darkerGrey;
-const SELECTED_ICON_COLOR = colors.yellow900;
-
-const LEFT_PADDING = 185;
+const DEFAULT_MENU_THEME: MenuTheme = {
+    paddingLeft: 30,
+    textColor: colors.white,
+    iconColor: colors.white,
+    selectedIconColor: colors.white,
+    selectedBackgroundColor: colors.menuItemDefaultSelectedBackground,
+};
 
 export const Menu: React.StatelessComponent<MenuProps> = (props: MenuProps) => {
     return (
-        <div style={{ paddingLeft: LEFT_PADDING }}>
-            {_.map(menuItemEntries, entry => {
+        <div>
+            {_.map(props.menuItemEntries, entry => {
                 const selected = entry.to === props.selectedPath;
                 return (
-                    <MenuItem key={entry.to} className="py2" to={entry.to}>
-                        <MenuItemLabel title={entry.labelText} iconName={entry.iconName} selected={selected} />
+                    <MenuItem key={entry.to} to={entry.to}>
+                        <MenuItemLabel
+                            title={entry.labelText}
+                            iconName={entry.iconName}
+                            selected={selected}
+                            theme={props.theme}
+                        />
                     </MenuItem>
                 );
             })}
         </div>
     );
 };
+Menu.defaultProps = {
+    theme: DEFAULT_MENU_THEME,
+    menuItemEntries: defaultMenuItemEntries,
+};
 
 interface MenuItemLabelProps {
     title: string;
     iconName: string;
     selected: boolean;
+    theme: MenuTheme;
 }
 const MenuItemLabel: React.StatelessComponent<MenuItemLabelProps> = (props: MenuItemLabelProps) => {
     const styles: Styles = {
-        iconStyle: {
-            color: props.selected ? SELECTED_ICON_COLOR : DEFAULT_ICON_COLOR,
+        root: {
+            backgroundColor: props.selected ? props.theme.selectedBackgroundColor : undefined,
```

Comment:
So do we use `undefined` instead of `null` in our code? I can get onboard with that but I am more accustomed with using null. The convention I've always followed is that you use null to express "the value for none" and undefined to express "this was never defined". I think this is an age-old question in JS and as long as we're consistent I'm happy to go with undefined.

Reply:
we pretty much use `undefined` everywhere except for conditional rendering of react components: https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering